site/content/docs/5.2/components/progress.md

```diff
@@ -110,6 +131,13 @@ Use background utility classes to change the appearance of individual progress b
 
 Include multiple progress bars in a progress component if you need.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant should not be used because it does not respect the Orange Design System specifications.
```

Comment:
So how do we do that at Orange? Seems to be a very common use case. Linked to one of the previous comment where it says that we can only use the primary color. Not probably to be treated here but designers need to find a solution.

Reply:
The need you're talking about is for Progress bar or for Data display (like gauges or Horizontal stacked bar chart)? The usage is different.

Reply:
We need to talk about it. Got the feeling that's the same.

Reply:
Any update ?

Reply:
Answer:  
Use cases examples: if you want to display the evolution of a vote in real time until it reaches the end time.
It could also be used to display the distribution of the rights of a product; until 30% are free to use and after you need to pay. In this case, it would be more a static progress bar.
Does this component belong to data display components or not ? it is not exactly the same thing as a stacked horizontal bar chart even if the rendering is close.
=> We keep the design callout message the way it is written.